.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: Pipeline
+on: [push]
+jobs:
+  tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    services:
+      redis:
+        image: redis
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.7, 3.0, 3.1]
+        gemfile:
+          - Gemfile
+          - gemfiles/rails_5_sidekiq_5.gemfile
+#          - gemfiles/rails_5_sidekiq_6.gemfile
```

Comment:
These tests fail, I've created https://invoca.atlassian.net/browse/TECH-10254 to address these issues.